diff --git a/include/js/Inventory.js b/include/js/Inventory.js
index 1c1e83e..4c511d1 100644
--- a/include/js/Inventory.js
+++ b/include/js/Inventory.js
@@ -1230,6 +1230,9 @@ function InventorySelectAll(mod,image_pth)
 					var desc = prod_array['desc'];
 					var subprod_ids = prod_array['subprod_ids'];
 					if(y>0) {
+						if (mod=='Receiptcards')
+						var row_id = window.opener.fnAddProductRowRI(mod,image_pth);
+						else
 						var row_id = window.opener.fnAddProductRow(mod,image_pth);
 					} else {
 						var row_id = prod_array['rowid'];
